pkg/reconciler/eventtype/eventtype.go

```diff
@@ -62,25 +53,5 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, et *v1beta2.EventType) p
 	}
 	et.Status.MarkBrokerExists()
```

Comment:
should we update the lifecycle methods to reflect that it is no longer a `Broker` but now a more generic `Reference`?

Reply:
https://github.com/matzew/eventing/tree/update_status_to_referene_instead_of_just_broker

this branch is independent of the change here, hence I separated. 

but I may merge both branches.